crates/next-api/src/pages.rs

```diff
@@ -693,15 +706,86 @@ impl PageEndpoint {
         Ok(Vc::upcast(page_loader))
     }
 
+    #[turbo_tasks::function]
+    async fn compute_internal_ssr_chunk_module(
```

Comment:
I would omit the `compute_` prefix, since we usually don't add that.

compute sounds like it would actually compute it when calling the function, but that might not be the case due to caching. So we usually call methods like getters.